### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by garlinhs**

### CODE REVIEW

File: READ0105.txt [hunk 1]
```diff
@@ -23,4 +23,4 @@ https://docs.google.com/document/d/1SipNvEDdPKj-y0f8mU8-vbwYyOUtohnCZMwos2kFEaI/
 
 
 
-Hi did you buy the gift?
```
Comment (rainbow1223) [inline READ0105.txt]: Hmm, That was very nice experience, I asked to your client what she want.

Comment (garlinhs) [reply READ0105.txt]: So, what did she want?

Comment (rainbow1223) [reply READ0105.txt]: I think the client require red money pocket. haha

Comment (garlinhs) [reply READ0105.txt]: :O

Comment (garlinhs) [reply READ0105.txt]: That's good

Comment (rainbow1223) [reply READ0105.txt]: Hmm.

**Pull request closed without merging**